tests/unit/agas/components/server/managed_refcnt_checker.cpp

```diff
@@ -47,7 +47,7 @@ managed_refcnt_checker::~managed_refcnt_checker()
         strm << ( boost::format("[%1%/%2%]: triggering flag %3%\n")
                 % prefix_ % this_ % target_);
 
-        hpx::trigger_lco_event(target_);
+        hpx::trigger_lco_event(target_, naming::address());
     }
```

Comment:
I'd like to leave the old API in place here as well. Having to pass a default constructed address() is confusing.